internal/provider/file/file_test.go

```diff
@@ -153,13 +157,35 @@ var (
 func TestListJobFilename(t *testing.T) {
 	fc := file.New(&model.PrdFile{
 		Filename: "./fixtures/dockerhub.yml",
-	})
-	assert.Equal(t, dockerhubFile, fc.ListJob())
+	}, defaultImageDefaults)
+
+	expectedFiles := dockerhubFile
+	actualFiles := fc.ListJob()
+	assert.Equal(t, len(expectedFiles), len(actualFiles))
+	for i, expected := range expectedFiles {
+		assert.EqualExportedValues(t, expected, actualFiles[i])
+	}
```

Comment:
Not sure why you need this change?

`assert.Equal(t, dockerhubFile, fc.ListJob())` should be enough.

Reply:
That fails due to the differing unexported value.

Reply:
Oh, and `EqualExportedValues` doesn’t accept slices.